app/server/awsIntegration.ts

```diff
@@ -12,12 +12,12 @@ const PROFILE = "membership";
 
 const CREDENTIAL_PROVIDER = new AWS.CredentialProviderChain([
   () => new AWS.SharedIniFileCredentials({ profile: PROFILE }),
-  ...AWS.CredentialProviderChain.defaultProviders
+  ...AWS.CredentialProviderChain.defaultProviders,
```

Comment:
I feel like these have made it in due to a `no-verify` commit at some point and thus your local prettier (which has different rules) has added extra/noisy changes to this PR... I recommend...
1) removing this file from the commit (since the only changes look to be incorrect prettier ones from your machine)
2) once this PR is out, addressing https://github.com/guardian/manage-frontend/wiki/Major-TODOs#move-prettier-config-to-a-config-file-for-consistency